MekHQ/src/mekhq/campaign/market/PartsStore.java

```diff
@@ -511,7 +511,8 @@ private void stockArmor(Campaign c) {
 		/*
 		 * Protomek Armor
 		 */
-		parts.add(new ProtomekArmor(0, 100, -1, true, c));
+        parts.add(new ProtomekArmor(0, EquipmentType.T_ARMOR_STANDARD, 100, -1, true, c));
```

Comment:
whitespace is a touch off

Reply:
That file had a mix of tabs and spaces. I fixed the indents.